### PR TITLE
[INFRA-4096] Bump node dependancies

### PR DESCRIPTION
**Pull request by zombar**
Update packages and lockfile

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,102 +31,102 @@
...
+    "express-async-handler": "1.2.0",
+    "express-validator": "^7.0.1",
+    "geoip-lite": "^1.4.8",
+    "helmet": "^7.1.0",
+    "ioredis": "^5.3.2",
+    "isbot": "^3.7.1",
+    "js-cookie": "^3.0.5",
+    "jsonwebtoken": "^9.0.2",
+    "jwt-decode": "^4.0.0",
+    "knex": "^3.0.1",
```
Comment (sazap10) [inline package.json:59]: This wont work for node 12 (current base image), v3 is compatible with node >=16. https://github.com/knex/knex/releases/tag/3.0.0

**Review by sazap10 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (zombar) [reply package.json:59]: Could we / should we bump it to node 21?

Comment (sazap10) [reply package.json:59]: probably bump to whatever is the latest LTS, i think that 18

**Comment by sazap10** [comment]
closing in favour of https://github.com/bugsnag/kutt/pull/10

**Pull request closed without merging**